frontends/web/src/locales/en/app.json

```diff
@@ -1277,7 +1277,8 @@
       "fee": "Enable custom fees",
       "setProxyAddress": "Set proxy address",
       "title": "Expert settings",
-      "useProxy": "Enable tor proxy"
+      "useProxy": "Enable tor proxy",
+      "useSats": "Use SATS unit for Bitcoin Accounts"
```

Comment:
in the app we use "title case" everywhere, i.e. no capitalized words in the middle of a sentence => "Bitcoin accounts".

Reply:
:+1: